lib/trento/hosts.ex

```diff
@@ -140,13 +140,20 @@ defmodule Trento.Hosts do
   end
 
   @spec request_operation(atom(), String.t(), map()) :: {:ok, String.t()} | {:error, any}
-  def request_operation(:saptune_solution_apply, host_id, params) do
+  def request_operation(operation, host_id, params)
+      when operation in [:saptune_solution_apply, :saptune_solution_change] do
     operation_id = UUID.uuid4()
 
+    operator =
```

Comment:
nitpick:
could we maybe have a `map_operation` kind of function?
we will continue adding more operation, and it will scale better

Reply:
Agreed. The liked PR should cover this